### PR TITLE
Fix Issue #1352 & Fix Issue #1355

### LINKED ISSUE #1355: Introduce Sorting of Cards in Meshery Catalog
Introduce sort by option as a dropdown in Meshery Catalog, which arranges the cards alphabetically and on the basis of time of update, both in ascending and descending manner.  

For reference take a look at [meshery.layer5.io/catalog](https://meshery.layer5.io/catalog)

### Screenshots / Mockups

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Sranjan0208**
**Description**

This PR fixes #1355 

Added Sort menu in catalog page. Also added margin-bottom to "Join Community" button in calendar page.

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -14,6 +14,9 @@ GEM
     ffi (1.15.5-x64-mingw32)
     forwardable-extended (2.6.0)
     google-protobuf (3.21.12)
+    google-protobuf (3.21.12-x64-mingw32)
+    google-protobuf (3.21.12-x86_64-darwin)
+    google-protobuf (3.21.12-x86_64-linux)
     http_parser.rb (0.8.0)
```
Comment (Chadha93) [inline Gemfile.lock:20]: @Sranjan0208 Could you please revert these?

Comment (Sranjan0208) [reply Gemfile.lock]: @Chadha93 It's done.

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@Sranjan0208, thanks much,  the initial work looks good
1. Update the location to the top-right, keep the sorting option above the filters, in the mobile view
2. Also, introduce padding around the element.
2. We also need to sort the cards on the basis of the last modified date 
3. Please revert the formatting changes, it makes the work of the reviewer more time-consuming and is undesirable until specifically asked.

**Comment by Sranjan0208** [comment]
@thisiskaransgit  Do you want "Updated At:" to be the basis of last modified date?

image

**Comment by vishalvivekm** [comment]
@Sranjan0208 Let's discuss it on the websites call on Monday. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by thisiskaransgit** [comment]
> @thisiskaransgit Do you want "Updated At:" to be the basis of last modified date?

Yes @Sranjan0208, this is the evident basis of sorting the cards

**Comment by Chadha93** [comment]
@Sranjan0208 Heads up merge conflict....

**Comment by Sranjan0208** [comment]
@thisiskaransgit  Incorporated all the changes you asked for

**Comment by iArchitSharma** [comment]
@Sranjan0208, why is there no sorting actually taking place when a sort option is selected?

**Comment by abhijeetgauravm** [comment]
Hey @Sranjan0208 Let's discuss this on website call today at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima)

**Review by abhijeetgauravm — CHANGES_REQUESTED** [review]
@Sranjan0208 sorting not working you may check in preview!

**Comment by iArchitSharma** [comment]
@abhijeetgauravm I guess we have to reassign the issue!

**Comment by abhijeetgauravm** [comment]
Closing this since there have been no updates from @Sranjan0208 for a long time!

**Pull request closed without merging**